src/platform/graphics/shader-chunks/frag/shared-wgsl.js

```diff
@@ -0,0 +1,17 @@
+export default /* glsl */`
+
+// convert clip space position into texture coordinates for sampling scene grab textures
```

Comment:
"scene grab textures"? What are those?

Reply:
color grab and depth grab textures

Reply:
this is existing webgl shader chunk, just translated to wgsl